python_modules/dagster-graphql/dagster_graphql/client/client.py

```diff
@@ -383,17 +383,23 @@ def shutdown_repository_location(
         else:
             raise Exception(f"Unexpected query result type {query_result_type}")
 
-    def terminate_run(self, run_id: str):
+    def terminate_run(self, run_id: str, force: bool = False):
         """Terminates a pipeline run. This method it is useful when you would like to stop a pipeline run
         based on a external event.
 
         Args:
             run_id (str): The run id of the pipeline run to terminate
+            force (bool, optional): if false, run will be terminated using terminatePolicy SAFE_TERMINATE.
```

Comment:
hm; not a huge fan of hiding terminatepolicy behind a flag like this. What if we down the road, add an additional terminatepolicy value? Seems like it would be hard to extend support here. How about instead we just take terminatePolicy directly?